onadata/libs/utils/export_builder.py

```diff
@@ -542,6 +565,7 @@ def write_row(row, csv_writer, fields):
                             self.pre_process_row(child_row, section),
                             csv_writer, fields)
             index += 1
+            track_task_progress(i, total_records)
```

Comment:
Would it be better to make this updates in chunks/batches of 10 or 100 or X? Something configurable in settings. Could be an overkill when you are iterating over 10k or 100k records.